<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="cpu_view">
       <attribute name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/icons/images/processor-cpu.png</normaloff>:/icons/images/processor-cpu.png</iconset>
       </attribute>
       <attribute name="title">
        <string>CPU</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="cpu_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="maps_view">
       <attribute name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/icons/images/memory-map.png</normaloff>:/icons/images/memory-map.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Maps</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="map_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actioncontinue"/>
    <addaction name="actionstop"/>
    <addaction name="actionmaps"/>
    <addaction name="actionregs"/>
    <addaction name="actionclose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionopen"/>
   <addaction name="actioncontinue"/>
   <addaction name="actionstop"/>
   <addaction name="actionmaps"/>
   <addaction name="actionregs"/>
   <addaction name="actionclose"/>
  </widget>
  <action name="actionopen">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icons/images/attach.png</normaloff>:/icons/images/attach.png</iconset>
   </property>
   <property name="text">
    <string>attach</string>
   </property>
  </action>
  <action name="actioncontinue">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icons/images/arrow-run-cursor.png</normaloff>:/icons/images/arrow-run-cursor.png</iconset>
   </property>
   <property name="text">
    <string>continue</string>
   </property>
  </action>
  <action name="actionstop">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icons/images/control-stop.png</normaloff>:/icons/images/control-stop.png</iconset>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </action>
  <action name="actionmaps">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icons/images/memory-map.png</normaloff>:/icons/images/memory-map.png</iconset>
   </property>
   <property name="text">
    <string>maps</string>
   </property>
  </action>
  <action name="actionregs">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icons/images/processor-cpu.png</normaloff>:/icons/images/processor-cpu.png</iconset>
   </property>
   <property name="text">
    <string>regs</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icons/images/close-all-tabs.png</normaloff>:/icons/images/close-all-tabs.png</iconset>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
